app/src/main/java/org/secuso/privacyfriendlyactivitytracker/activities/HelpActivity.java

```diff
@@ -51,10 +54,15 @@ protected void onCreate(Bundle savedInstanceState) {
         generalExpandableListView.setAdapter(expandableListAdapter);
 
         overridePendingTransition(0, 0);
+
+        getWindow().getDecorView().setLayoutDirection(View.LAYOUT_DIRECTION_RTL);
```

Comment:
this is not necessary .. the layout will just adjust itself if edited propperly. This should not be set by default.